.github/workflows/tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       matrix:
         build_type: [backend, frontend, system, annotations]
         target: [core, plugins]
-        ruby: ['3.1']
+        ruby: ['3.2']
```

Comment:
Finding it hard to read the `container: ` definition. Bear with me, this is: 

- switching to testing with 3.2 (matrix `ruby: ['3.2']`)
- container definition will append `-ruby-3.1.0` only if a buid has ruby set to 3.1, which we are no longer using, so should that container suffix still be there?

Reply:
We are already testing on 3.2, this PR only updates the string so it is in sync with what is actually happening

Reply:
Right, but do we still need `${{ (matrix.ruby == '3.1') && '-ruby-3.1.0' || '' }}`?